src/interface/src/app/plan/summary-panel/summary-panel.component.html

```diff
@@ -0,0 +1,37 @@
+<div class="summary-panel">
+  <div class="summary-header">
+    <div class="summary-title">{{summaryInput?.type}} Summary</div>
+    <div class="summary-subtitle">Status: {{summaryInput?.status}}</div>
+  </div>
+  <div class="summary-content">
+    <div>Owner: {{summaryInput?.owner}}</div>
+    <div>Created: {{summaryInput?.createdTime}}</div>
+    <div>Last updated: {{summaryInput?.updatedTime}}</div>
+    <div>Region: {{summaryInput?.region}}</div>
+
+    <div class="grid-container">
```

Comment:
totally optional, but you could look into https://material.angular.io/components/grid-list/overview to maybe have an easier time displaying these grid items

Reply:
thanks! I saw mat-grid-list after making the grid with css haha.. I think it's simple enough that we don't need it right now